src/components/Summary/AlterationSummary.vue

```diff
@@ -232,6 +238,23 @@ export default class AlterationSummary extends Mixins(
   /** Prop to perform validation. */
   @Prop() readonly validate: boolean
 
+  /** The future effective fee for Alteration. */
+  futureEffectiveFees: number = 0
+
+  async mounted () {
+    // fetches the fee prices to display in the text
+    const feePrices = await this.fetchFeePrices()
```

Comment:
Why do you need this, since you already did it in Alteration.vue right? Can you just read the values that you saved to the store?

Reply:
I'm not saving the feePrices in the store, only the currentFees for the filing. Do you think I should save it to the store as well?

Reply:
The problem I see is too many calls that do (almost) the same thing.

You could possibly just make 1 call, in Alteration.vue, and then extract/sum the individual fees you care about. Unfortunately that's making an assumption about adding the individual fees... but I think I saw somewhere in here where you already do that, so it's not any worse :)

Reply:
ok. I added the feePrices to the store. Thanks.